### PR TITLE
Revised the README doc for readability

### PR DESCRIPTION
**Pull request by rajkaramchedu**
@JanuszL I reviewed and made a few changes to the README doc. I think these will enhance the readability. See what you think. Here is the Gist preview with my changes: https://gist.github.com/rajkaramchedu/67af24cd19a683dfb0e030cc6f6a232f

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -2,37 +2,46 @@
...
+* Support for image classification and segmentation workloads.
+* Ease of integration through direct framework plugins and open source bindings.
+* Portable training workflows with multiple input formats - JPEG, PNG (fallback to CPU), TIFF (fallback to CPU), BMP (fallback to CPU), raw formats, LMDB, RecordIO, TFRecord.
+* Extensible for user-specific needs through open source license.
 
 .. overview-end-marker-do-not-remove
 
 .. installation-begin-marker-do-not-remove
 
+---
```
Comment (JanuszL) [inline README.rst]: What is the purpose of this? It doesn't look well.
If you wanted to make separating line I believe it is `----`

File: README.rst [hunk 2]
```diff
@@ -73,61 +111,72 @@ Prerequisites
...
-   |      -  |tf link|_ or later                                                                                                          |
+   |      * |mxnet link|_ ``mxnet-cu90`` or later                                                                                         |
+   |      * |pytorch link|_                                                                                                               |
+   |      * |tf link|_ or later                                                                                                           |
    +----------------------------------------+---------------------------------------------------------------------------------------------+
 
-.. note::
 
-   TensorFlow installation is required to build the TensorFlow plugin for DALI
+**NOTE:**
```
Comment (JanuszL) [inline README.rst]: Do we want to replace note with just formatted string?

File: README.rst (hunk 1 above)
Comment (rajkaramchedu) [reply README.rst]: Fixed.

File: README.rst (hunk 2 above)
Comment (rajkaramchedu) [reply README.rst]: I went back to the note admonition. (I was considering the formatted string because GitHub doesn't appear to support the admonition. )

**Pull request merged**